tests/foreman/ui/test_host.py

```diff
@@ -868,6 +868,89 @@ def test_positive_search_by_parameter(session, module_org, smart_proxy_location,
         assert values[0]['Name'] == param_host.name
 
 
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_match('8')
+@pytest.mark.no_containers
```

Comment:
we can run test without this marker, can you try it?
```suggestion
```

Reply:
Fixed, PRT passed.

Reply:
@vijaysawant Hi, just FYI, I had to put the `no_containers` marker back, because without it the latest PRTs started to fail with a Docker error
```
failed on setup with "docker.errors.APIError: 409 Client Error for http+docker://ssh/v1.43/containers/91a0033c33681a28debc78856f39b2dccc98fb2c735d8503dcbdd94763ca5b1e/exec: Conflict ("Container 91a0033c33681a28debc78856f39b2dccc98fb2c735d8503dcbdd94763ca5b1e is not running")"
```
With the marker back, PRT passed and the test uses a "real" host, not a container.